Contrib/CLI/wcli.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+function config_extract() {
+    echo $(cat ~/.walletwasabi/client/Config.json | jq -r "$1")
```

Comment:
`echo` and `cat` is not needed here.
```suggestion
    jq -r "$1" ~/.walletwasabi/client/Config.json
```